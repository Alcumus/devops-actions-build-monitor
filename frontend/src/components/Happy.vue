<template>
  <div class="container">
    <img
      :src="happyImage"
      alt="no-failures"
    >
  </div>
</template>

<script>
import happy from '../images/happy.png';

export default {
  name: "Happy",
  data() {
    return {
      happyImage: happy
    }
  }

}
</script>

<style scoped>
.container {
  height: 70%;
  display: flex;
  justify-content: center;
  align-items: center;
}

img {
  height: 40%;
}
</style>
