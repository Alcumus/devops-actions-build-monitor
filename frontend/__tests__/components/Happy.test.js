import { describe, it, expect } from 'vitest';
import { mount } from '@vue/test-utils';
import Happy from '@/components/Happy';

describe('Happy', () => {
  it('should render happy image', () => {
    expect(mount(Happy).html()).toMatchSnapshot();
  });
});
