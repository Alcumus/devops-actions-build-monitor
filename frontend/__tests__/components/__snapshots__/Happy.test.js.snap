// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Happy > should render happy image 1`] = `
<div class="container">
  <img alt="no-failures" src="/src/images/happy.png">
</div>
`;
